src/components/fields/date-input/date-input.tsx

```diff
@@ -59,7 +65,27 @@ export const DateInput = (props: IGenericFieldProps<IDateInputSchema>) => {
 				.test("not-past", notPastRule?.["errorMessage"] || ERROR_MESSAGES.DATE.CANNOT_BE_PAST, (value) => {
 					if (!isValidDate(value) || !notPastRule?.["notPast"]) return true;
 					return !LocalDate.parse(value).isBefore(LocalDate.now());
-				}),
+				})
+				.test(
+					"min-date",
+					minDateRule?.["errorMessage"] ||
+						ERROR_MESSAGES.DATE.MIN_DATE(
+							DateTimeHelper.formatDateTime(minDateRule?.["minDate"], "dd/MM/uuuu", "date")
+						),
+					(value) => {
+						if (!isValidDate(value) || !minDate) return true;
+						return !LocalDate.parse(value).isBefore(minDate);
+					}
+				)
+				.test(
+					"max-date",
+					maxDateRule?.["errorMessage"] ||
+						DateTimeHelper.formatDateTime(maxDateRule?.["maxDate"], "dd/MM/uuuu", "date"),
```

Comment:
missing the max date error message

Reply:
whoops my bad, I missed that out, I have just added